backend/python/app/services/implementations/story_service.py

```diff
@@ -153,15 +153,13 @@ def get_story_translation(self, id):
 
     def assign_user_as_reviewer(self, user, story_translation):
         if (
-            story_translation.language in user.approved_languages
```

Comment:
`assignUserAsReviewer` always returns `User can't be assigned as a reviewer`. Perhaps the conditional checks on mysql JSONs are different? Maybe one of these lines has something to do with this regression? 
Testing this takes some data to set up, so lmk if you want to pair program with me since I have it all on my local db.